compiler.rst

```diff
@@ -209,29 +201,29 @@ one is working with (e.g., if you are working with an 'if' statement
 you need to watch out for the ':' token to find the end of the conditional).
 
 The functions called to generate AST nodes from the parse tree all have
-the name ast_for_xx where xx is the grammar rule that the function
-handles (alias_for_import_name is the exception to this).  These in turn
+the name ``ast_for_xx`` where xx is the grammar rule that the function
```

Comment:
Emphasize xx: `*xx*`. And in other cases below.

Maybe ast_for_xx can be formatted as: \`\`ast_for_\`\`\*xx\*. But I don't know whether this works and how it looks.